bigquery_etl/cli/backfill.py

```diff
@@ -233,20 +234,26 @@ def create(
     ./bqetl backfill validate
     """
 )
-@click.argument("qualified_table_name", required=False)
+@click.argument("asset_name", required=False)
 @sql_dir_option
 @project_id_option()
 @ignore_missing_metadata_option
 @click.pass_context
 def validate(
     ctx,
-    qualified_table_name,
+    asset_name,
     sql_dir,
     project_id,
     ignore_missing_metadata,
 ):
     """Validate backfill.yaml files."""
-    if qualified_table_name:
+    if asset_name:
+        # checking if path instead of "dataset.table_name" format was passed
```

Comment:
IMO allowing multiple input types is more confusing and trying to coerce a path into a name might cause unexpected errors.  e.g. if you did `bqetl validate sql/project/dataset/table` without a file then the line below would give `sql.project.dataset`

I think it would be better to try to detect if it's a path and raise an error that says something like `Expected qualified table name (project.dataset.table_name), not a path: {qualified_table_name}`

Reply:
I think the problem here is that pre-commit would pass in the file path. But perhaps that is wrong and there is a way to modify what the command input would be. Honestly, I haven't really worked with pre-commit hooks that much but I do think it would be useful to have this check and have it run at this stage instead of waiting for the CI to tell me this 20 minutes later.

I guess another option could be to include an optional parameter to the command which accepts path, WDYT?

Reply:
ok, reverted the argument name change and added additional comment for the context of this change / logic